deploy/docker/README.md

```diff
@@ -39,23 +39,29 @@ The `lfedge/ekuiper` images come in many flavors, each designed for a specific u
 
 This is the development Docker image, which is based on Debian and it also includes a Golang build environment. If you are unsure about what your needs  are, you probably want to use this one. It is designed to be used both as a throw away container (mount your source code, compile plugins for eKuiper,  and start the  container to run your app), as well as the base to build other images. Please be aware of that this image is the biggest size, and it's usually used for development purpose.
```

Comment:
Here we should highlight on current status which the default tag is alpine based